seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -485,6 +485,7 @@ angular.module('BE.seed.controller.inventory_list', [])
       $scope.columns.unshift({
         name: 'merged_indicator',
         displayName: '',
+        headerCellTemplate: '<span></span>', // remove header
```

Comment:
Visually, it didn't seem like there was a default that needed to be removed (here and for the other `headerCellTemplate` as well), but things seem to work as expected.

Reply:
I did that just to make it obvious that the others aren't clickable (cursor == `default`), previously the cursor showed as `pointer` when you hover over them making you think they might be sortable